packages/common/tsconfig.json

```diff
@@ -3,6 +3,7 @@
     "outDir": "dist",
     "baseUrl": ".",
     "paths": {
+      "audius-query": ["src/audius-query/index.ts"],
```

Comment:
what does this do?

Reply:
makes absolute imports possible, which i kinda like with audius-query cause it's kinda like it's own package

Reply:
OH that's cool